data/rulei/rulei.txt

```diff
@@ -90,6 +90,23 @@ mission "Rulei Planet"
 			`	It's best you don't remain here for too long.`
 				decline
 
+
+mission "Umbral's Reach"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		conversation
+			`Out of nowhere, the mysterious whispers of the Rulei seep into your mind once more. They murmur slowly at first, but grow faster and louder from the direction of the ancient vessel, eventually coalescing into one spoken phrase.`
+			`	"Beware..."`
+			`	"...Umbral's Reach."`
```

Comment:
Does this have something to do with the system Umbral (connected to Tarazed), or are they unrelated?

Reply:
No. "Umbral Reach" is the name for the dark region that was decided after a bit of discussion with others. Void, Abyss, etc. will be used in Mata content as far as I know, and anything else we came up didn't really fit it. Perhaps I reword this back to "Umbral Reach" so it doesn't possibly imply that "Umbral is reaching", as in the planet?

Reply:
That sounds good.
```suggestion
			`	"...Umbral Reach."`
```